ergotree-interpreter/src/sigma_protocol/unchecked_tree.rs

```diff
@@ -116,6 +130,12 @@ impl UncheckedSchnorr {
     pub fn with_challenge(self, challenge: Challenge) -> Self {
         UncheckedSchnorr { challenge, ..self }
     }
+    /// compute commitment
+    pub fn compute_commitment(&self) -> FirstDlogProverMessage {
```

Comment:
1. Why do we need to compute commitments here and for DH tuple below and not use `self.commitmentOpt`?
2. Why `verifier.compute_commitments()` call is missing in `bag_for_multisig` after https://github.com/sahandzou/sigma-rust/blob/7b1aeea527bc05a24d0c868fe24b5b7f6f14aa54/ergo-lib/src/wallet/multi_sig.rs#L121 ?